package.json

```diff
@@ -40,7 +40,7 @@
   "devDependencies": {
     "@types/algoliasearch": "^3.30.12",
     "algolia-frontend-components": "0.0.35",
-    "algoliasearch": "3.32.0",
+    "algoliasearch": ">= 3.32.0 < 5",
```

Comment:
The dev dependency should be pinned to `4.0.0-beta.14`.

Reply:
Why buddy?

Reply:
For consistency. We always want a reproducible development environment, which is why we pin all dev dependencies.

You can read more on the [Renovate website](https://docs.renovatebot.com/dependency-pinning/).